app/templates/views/admin/gallery_edit.html

```diff
@@ -6,38 +6,37 @@
 {% if gallery is defined %}
 action="{{url_for('administrator.gallery_edit', gid=gallery.gid)}}"
 {% else %}
-action="{{url_for('administrator.gallery_add')}}"
-{% endif %} method="post">
-
-  <div class="row">
-      <div class="form-group">
-        <label for="galleryName">Gallery</label>
-        <input type="text"
-          id="galleryName"
-          class="form-control mx-sm-3"
-          aria-describedby="gallery title"
-          name="title"
-          value="{% if gallery is defined%}{{gallery.title}}{% else %} {% endif %}">
-      </div>
-  </div>
-
-  <div class="row">
-
-    <div class="col-sm-3">
-      <div class="form-group">
-        <div class='input-group date' id='open_date'
-        {% if gallery is  defined %}
-        value="{{gallery.open_date}}"
-        {% else %}
-        value=""
-        {% endif %}>
-          <input type='text' class="form-control" name="open_date"/>
-          <span class="input-group-addon">
-            <span class="glyphicon glyphicon-time"></span>
-          </span>
-        </div>
+action="{{url_for('gallery_add')}}"
```

Comment:
Note: Blueprint requires that routes be defined using the @admin or @public routes. This changes how url_for works so instead of gallery_add, the parameter needs to be administrator.gallery_add or public.<function>,

I got this, I am merging it in but side note